src/energyplus/ForwardTranslator/ForwardTranslateZoneHVACPackagedTerminalAirConditioner.cpp

```diff
@@ -317,6 +318,9 @@ namespace energyplus {
       if (_coolingCoil->iddObject().type() == IddObjectType::Coil_Cooling_DX_SingleSpeed) {
         _coolingCoil->setString(Coil_Cooling_DX_SingleSpeedFields::AirInletNodeName, coolingCoilInletNodeName);
         _coolingCoil->setString(Coil_Cooling_DX_SingleSpeedFields::AirOutletNodeName, coolingCoilOutletNodeName);
+      } else if (_coolingCoil->iddObject().type() == IddObjectType::Coil_Cooling_DX_SingleSpeed_ThermalStorage) {
+        _coolingCoil->setString(Coil_Cooling_DX_SingleSpeed_ThermalStorageFields::EvaporatorAirInletNodeName, coolingCoilInletNodeName);
+        _coolingCoil->setString(Coil_Cooling_DX_SingleSpeed_ThermalStorageFields::EvaporatorAirOutletNodeName, coolingCoilOutletNodeName);
```

Comment:
Looks like this one does a generic `translateAndMapModelObject(coolingCoil)`...

Reply:
Right, I just assumed it was the same as the PTHP one.

Reply:
Either we can assign it to a PTAC, PTHP, then we should have a containingZoneHVACComponent implemented in model, or this should be removed.

Either way, looks like we need model tests indeed to check what this can or cannot be assigned to.

Reply:
After a quick search on the proposed IDD, it looks like this coil can only be on `AirLoopHVAC:UnitarySystem` and `CoilSystem:Cooling:DX`. I was wrong about the ZoneHVAC and other HVAC...